src/drivers/mct/cime_config/config_component.xml

```diff
@@ -2313,7 +2313,7 @@
 
   <entry id="PIO_VERSION">
     <type>integer</type>
-    <default_value>1</default_value>
+    <default_value>2</default_value>
```

Comment:
This is the only change outside of the pio2 directory

Reply:
Actually I also ended up with a change in the Makefile to get mpi-serial working.